script/gitlabci/job_base.yml

```diff
@@ -4,6 +4,9 @@
     ALPAKA_CI_BOOST_BRANCH: "boost-${ALPAKA_BOOST_VERSION}"
     BOOST_LIBRARYDIR: "/opt/boost/${ALPAKA_BOOST_VERSION}/lib"
     ALPAKA_CI_OS_NAME: "Linux"
+    # $CI_CPUS is defined by the HZDR gitlab CI
+    ALPAKA_CI_BUILD_JOBS: "$CI_CPUS"
+    OMP_NUM_THREADS: "$CI_CPUS"
```

Comment:
So `CI_CPUS` is the number of virtual cores? Funnily enough, GitHub claims they also have 2 cores on each runner, but LLAMA builds notably faster with 4 build jobs than 2. So I wondered if GitHub gives you 2 physical cores with hyperthreading. Anyhow, nothing wrong with your code here!

Reply:
> So CI_CPUS is the number of virtual cores?

Yes this is the number of virtual cores the CI instance can use. I have not checked if this is a suggestion to not interrupt other instances or if all other cores are hidden. I think this depends on the node. We have OpenStack virtualized nodes for CPU and native nodes for CUDA and HIP.